addon/components/polaris-text-field/resizer.js

```diff
@@ -0,0 +1,132 @@
+import Component from '@ember/component';
+import layout from '../../templates/components/polaris-text-field/resizer';
+import { htmlSafe } from '@ember/string';
+import { computed } from '@ember/object';
+import { scheduleOnce } from '@ember/runloop';
+
+const REPLACE_REGEX = /[\n&<>]/g;
+
+const ENTITIES_TO_REPLACE = {
+  '&': '&amp;',
+  '<': '&lt;',
+  '>': '&gt;',
+  '\n': '<br>',
+};
+
+const replaceEntity = (entity) => {
+  return ENTITIES_TO_REPLACE[entity] || entity;
+};
+
+export default Component.extend({
+  classNames: ['Polaris-TextField__Resizer'],
+
+  'aria-hidden': true,
```

Comment:
Not keen on this dasherized prop name, especially as it's not bound to anything on the DOM. Make this `ariaHidden` pls 👍

EDIT: if this is _meant_ to be bound to the resizer element, you need to add an attribute binding (which can accept the `ariaHidden` name anyway)

Reply:
This value doesn't seem to ever change so I don't think it requires binding, it just needs an initial value. I'll update to `ariaHidden`.

Reply:
Yeah, I wasn't thinking about the value changing, just that it's something I've done elsewhere to declare something like an `ariaHidden` property and apply it to the component's root element using `attributeBindings` 😉